arrow-core/src/main/java/com/raphtory/arrowcore/implementation/VertexHistoryPartition.java

```diff
@@ -140,16 +141,16 @@ public int compare(int row1, int row2) {
 
 
 
-    private final int _partitionId;
-    private final VertexPartition _avp;
-    private final VertexPartitionManager _apm;
+    protected final int _partitionId;
+    protected final VertexPartition _avp;
+    protected final VertexPartitionManager _apm;
     //private final VertexSnapshotPartition _snapshot;
 
-    private VertexHistoryStore _history;
-    private VectorSchemaRoot _historyRO;
-    private ArrowFileReader _historyReader;
-    private boolean _modified = false;
-    private boolean _sorted = false;
+    protected VertexHistoryStore _history;
+    protected VectorSchemaRoot _historyRO;
+    protected ArrowFileReader _historyReader;
+    protected boolean _modified = false;
+    protected boolean _sorted = false;
 
 
     /**
```

Comment:
//_snapshot = new VertexSnapshotPartition(partitionId, avp); needed below?

Reply:
Same with //_snapshot.initialize();

Reply:
And 
        if (updated) {
            //_snapshot.takeSnapshot();
        }

Reply:
//System.out.println("CLOSING PARTITION: " + partitionId);